packages/workspace-store/src/navigation/helpers/traverse-paths.ts

```diff
@@ -44,6 +44,8 @@ const createOperationEntry = (
   }
 }
 
+const methods = new Set(['get', 'put', 'post', 'delete', 'patch', 'connect', 'options', 'head', 'trace'])
```

Comment:
We actually have a set already if u want it

https://github.com/scalar/scalar/blob/main/packages/helpers/src/http/http-methods.ts#L8

Reply:
should probably freeze it haha